src/styles/utils/variables.scss

```diff
@@ -0,0 +1,13 @@
+$cardLogoWidth: 160px;
+$cardLogoHeight: 134px;
+$cardWidth: 166px;
+$lineHeitghtBig: 18px;
+$lineHeitghtMiddle: 16px;
+$lineHeitghtSmall: 14px;
```

Comment:
There is a typo in the variable name `$lineHeitghtSmall`. It should be `$lineHeightSmall` to follow the correct spelling of 'height'.